website/docs/cli-reference.md

```diff
@@ -53,6 +53,11 @@ Required.
 Path to compiled contract.
 #### `--account-address ADDRESS`
 Account address.
+#### `--block-explorer BLOCK_EXPLORER=starkscan`
+Generated links will point to that block explorer. Available values:
+- starkscan
+- viewblock
+- voyager
```

Comment:
I think users should be somehow aware that we can add one on their request (after verification, if the project is mature enough). We should either include it here or somewhere in the docs

Reply:
This issue was requested by users. I am not sure if it's needed.